rebar.lock

```diff
@@ -0,0 +1,6 @@
+{"1.1.0",
+[{<<"hex_core">>,{pkg,<<"hex_core">>,<<"0.2.1">>},0}]}.
```

Comment:
fyi the latest version is 0.7.1 now, consider upgrading just in case. We may have introduced minor incompatibilities in the meantime, sorry about that!